python/servo/build_commands.py

```diff
@@ -146,8 +146,9 @@ def build(
         # Gather Cargo build timings (https://doc.rust-lang.org/cargo/reference/timings.html).
         opts = ["--timings"] + opts
 
+        crown_enabled = "enabled" if kwargs.get("use_crown", False) else "disabled"
+        print(f"Building `{build_type.directory_name()}` build with crown {crown_enabled}.")
```

Comment:
I think it would be better to print some info/disclaimer when crown is not enabled. Something like `crown in not enabled, so rooting errors are not reported`.

Reply:
I put "(no JS garbage collection linting)" here as most people likely don't know what "rooting" means.